deployer/Cargo.toml

```diff
@@ -72,4 +72,6 @@ features = ["builder"]
 ctor = { workspace = true }
 hex = "0.4.3"
 rand = { workspace = true }
+shuttle-common-tests = { path = "../common-tests" }
```

Comment:
Maybe it should come from the workspace instead

Reply:
Yes, should be better thanks: [b3eab46](https://github.com/shuttle-hq/shuttle/pull/1204/commits/b3eab46e0dd5a7e3d3ceb31ed401a2ba6716ca0f).